website/venues/views.py

```diff
@@ -60,3 +62,69 @@ class ListReservationsView(ListView):
     def get_queryset(self):
         """Only allow access to user's own reservations."""
         return super().get_queryset().filter(user__pk=self.request.user.pk)
+
+
+class ReservationFeed(ICalFeed):
+    """Output an iCal feed containing all reservations."""
+
+    def product_id(self):
+        """Get product ID."""
+        return f"-//{Site.objects.get_current().domain}//EventCalendar//EN"
```

Comment:
I think this is supposed to be globally unique: https://www.kanzaki.com/docs/ical/prodid.html

So For the ReservationVenueFeed, I think we should make it a different product id.